go.mod

```diff
@@ -5,16 +5,16 @@ go 1.21
 require (
 	github.com/go-logr/logr v1.2.4
 	github.com/google/go-cmp v0.5.9
+	github.com/mitchellh/hashstructure v1.1.0
```

Comment:
looks like abandoned library. 

![Screenshot 2024-04-04 at 13-31-34 Tags · mitchellh_hashstructure](https://github.com/Kuadrant/limitador-operator/assets/881529/fa7ec818-7ec8-494d-9c93-b622ba971275)

![Screenshot 2024-04-04 at 13-32-34 mitchellh_hashstructure Get hash values for arbitrary values in Go (golang)](https://github.com/Kuadrant/limitador-operator/assets/881529/44075a2d-2421-4209-ab7a-3169c5c9a6dc)